src/main/java/Kafka.java

```diff
@@ -0,0 +1,136 @@
+        System.out.println("  Here are the tasks in your list:");
+        for (int i = 0; i < this.tasks.size(); i++) {
+            Task t = this.tasks.get(i);
+            String listMessage = "  " + (i + 1) + "." + t;
+            System.out.println(listMessage);
+        }
+    }
+
+    public void mark(int taskNumber) {
+        Task t = this.tasks.get(taskNumber - 1);
+        t.markAsDone();
+        String message = "  Good work on this task. Want a prize?:\n"
+                + "    " + t;
+        System.out.println(message);
+    }
+
+    public void unmark(int taskNumber) {
+        Task t = this.tasks.get(taskNumber - 1);
+        t.markAsNotDone();
+        String message = "  Hurry up. This task is necessary for Elio's script:\n"
+                + "    " + t;
+        System.out.println(message);
+    }
+
+    public void delete(int taskNumber) {
+        if (this.tasks.isEmpty()) {
+            return;
+        }
+        Task t = this.tasks.get(taskNumber - 1);
+        this.tasks.remove(taskNumber - 1);
+        String message = "  I've removed this task:\n"
+                + "    " + t;
+        System.out.println(message);
+        System.out.println("  Now you have " + this.tasks.size() + " task(s) in the list.");
+    }
+
+    public static void main(String[] args) {
+        String logo = """
+                   __  __            __     _
+                  |  |/  /  ____   _/  /_  | |      ____
+                  |     /  / _  | |_    _| | |/ /  / _  |
+                  |     \\ | |_| |   |  |   |   <  | |_| |
+                  |__|\\__\\ \\____|   |__|   |_|\\ \\  \\____|
+                """;
+        Scanner scanner = new Scanner(System.in);
+        Kafka kafka = new Kafka();
+        boolean isExitChat = false;
+
+        System.out.println("  Hello from\n" + logo);
+        kafka.greet();
+        System.out.println("  What do you need me for?");
+        while (!isExitChat) {
+            String[] userInput = scanner.nextLine().trim().split(" ", 2);
+            try {
+                if (userInput[0] == null) {
+                    continue;
+                }
+                if (userInput[0].equalsIgnoreCase("bye")) {
+                    isExitChat = true;
+                } else if (userInput[0].equalsIgnoreCase("list")) {
+                    kafka.createList();
+                } else if (userInput[0].equalsIgnoreCase("mark")) {
+                    int taskNumber = Integer.parseInt(userInput[1]);
+                    kafka.mark(taskNumber);
+                } else if (userInput[0].equalsIgnoreCase("unmark")) {
+                    int taskNumber = Integer.parseInt(userInput[1]);
+                    kafka.unmark(taskNumber);
+                } else if (userInput[0].equalsIgnoreCase("delete")) {
+                    int taskNumber = Integer.parseInt(userInput[1]);
+                    kafka.delete(taskNumber);
+                } else {
+                    if (userInput[0].equalsIgnoreCase("todo")) {
```

Comment:
Nested if statements are not as easy to read. Is it possible to just add more elif to handle the add task cases instead of nesting them in the else block?